spec/requests/api/tags_spec.rb

```diff
@@ -186,11 +186,14 @@
                      type: :object,
                      properties: {
                        id: {
-                         type: :string,
+                         type: :integer,
```

Comment:
Acknowledging here that this may break things but due to way our API is designed, or rather has never been designed, there is no way for us to deprecate things cleanly.